### PR TITLE
Added rdf and turtle urls

### PR DESCRIPTION
**Pull request by PaavoNykanen**
- Added redirect for turle and rdf files

### CODE REVIEW

File: ldasw/.htaccess [hunk 1]
```diff
@@ -17,5 +17,8 @@ RewriteCond %{HTTP_ACCEPT} text/\* [OR]
 RewriteCond %{HTTP_ACCEPT} \*/\* [OR]
 RewriteCond %{HTTP_ACCEPT} ^.*application/rdf\+xml.* 
 RewriteRule ^university$ https://raw.githubusercontent.com/PaavoNykanen/Ontologies/main/LinkedData/UniversityOntology/university.owl [R=303,NE,L]
+RewriteRule ^university$ https://raw.githubusercontent.com/PaavoNykanen/Ontologies/main/LinkedData/UniversityOntology/university.rdf [R=303,NE,L]
```
Comment (dgarijo) [inline ldasw/.htaccess:20]: This one is not reachable, as the previous one will trigger the rule

**Comment by dgarijo** [comment]
Please have a look at my small comment. Did you try the rules locally?

**Comment by PaavoNykanen** [comment]
> Please have a look at my small comment. Did you try the rules locally?

Sorry which comment? I didn't try this locally since I don't know how.

**Comment by dgarijo** [comment]
Now it should be vissible

File: ldasw/.htaccess (hunk 1 above)
Comment (PaavoNykanen) [reply ldasw/.htaccess:20]: Do you have a suggestion how to fix this? Maybe adding 'RewriteCond %{HTTP_ACCEPT} ^.*application/owl\+xml.* ' before the owl link and the the rdf condition for the rdf file?

Or should the owl or rdf be moved to the end as a default response?

Comment (dgarijo) [reply ldasw/.htaccess:20]: you have to choose on which file to return when you are requested application/rdf+xml

Comment (PaavoNykanen) [reply ldasw/.htaccess:20]: So if I left the ref+xml condition for the .rdf file, is there a similar condition for owl? Like application/owl+xml?

Comment (dgarijo) [reply ldasw/.htaccess:20]: OWL is not a serialization, right? I would leave only the rdf file. Or if you want to return an .owl file when people ask for rdf, then leave just the OWL file. It's up to you to decide what is being returned.

Comment (dgarijo) [reply ldasw/.htaccess:20]: well, `.rdf` is not a serialization either :)

**Pull request closed without merging**